core/src/main/java/com/exadel/etoolbox/backpack/core/services/pckg/DeletePackageService.java

```diff
@@ -0,0 +1,17 @@
+package com.exadel.etoolbox.backpack.core.services.pckg;
+
+import org.apache.sling.api.resource.ResourceResolver;
+
+/**
+ * Represents a service running in an AEM instance responsible for deleting package
+ */
+public interface DeletePackageService {
```

Comment:
(Possible improvement) Guys, is there a reason for having not a usual create-update-delete service but as many as five different services, one to create a package, one to edit, one to delete, etc.?

Reply:
may be not, we can use one